test/conftest.py

```
@@ -37,3 +42,98 @@ def fixture_video_embeddings_path(video_path: Path) -> Path:
+
+    # Start go2rtc
+    go2rtc_command = [
+        "./go2rtc",  # Adjust path to go2rtc executable if needed
+        "--config",
+        str(config_path),
+    ]
+
+    process = subprocess.Popen(go2rtc_command)
+    try:
+
+        def is_rtsp_stream_ready(rtsp_address):
```

Ruff (0.11.9)

Missing return type annotation for private function `is_rtsp_stream_ready`

Add return type annotation: `bool | None`

(ANN202)